core/src/apps/cardano/helpers/bech32.py

```diff
@@ -25,15 +24,12 @@ def encode(hrp: str, data: bytes) -> str:
 
 
 def decode_unsafe(bech: str) -> bytes:
-    hrp = get_hrp(bech)
-    return decode(hrp, bech)
+    # get_hrp
```

Comment:
this code is so straightforward that this comment is causing more confusion than helping, please remove it

Reply:
Deleted in [f38408b](https://github.com/trezor/trezor-firmware/pull/2525/commits/f38408bc5e551a04efb3b56a35ec4c2918b53137)